src/core_plugins/metrics/server/lib/vis_data/handle_error_response.js

```diff
@@ -1,5 +1,5 @@
 export default panel => error => {
-  console.log(error);
+  if (error.isBoom) throw error;
```

Comment:
At this point we only want to throw the 401 to breakout of the promise chain so the UI can show the auth dialog box. I will change this to check for the status 401.